src/Config/Config.php

```diff
@@ -139,7 +139,7 @@ public function offsetExists($offset): bool
 	 * @return mixed
 	 * @throws InvalidStateException
 	 */
-	public function offsetGet($offset)
```

Comment:
Well, this library support PHP >=7.2. Mixed typed as return type cannot be used.

Reply:
> or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in 

Just use annotation.